runtime/centrifuge/src/migrations.rs

```diff
@@ -181,57 +193,413 @@ mod asset_registry {
 		}
 	}
 
-	/// Register the LiquidityPools Wrapped Ethereum USDC
-	pub struct RegisterLpEthUSDC;
```

Comment:
Removing `RegisterLpEthUSDC` migration and adding it within the `AssetRegistryMultilocationToXCMV3` one